core/blockchain.go

```diff
@@ -464,6 +506,17 @@ func (bc *BlockChain) dispatchTxUnindexer() {
 		return
 	}
 	defer sub.Unsubscribe()
+	log.Info("Initialized transaction unindexer", "limit", txLookupLimit)
+
+	// XXX
```

Comment:
not sure why the test fails when I uncomment the below.
Not sure we need to take this change either.